src/main/java/com/googlesource/gerrit/plugins/oauth/HttpModule.java

```diff
@@ -27,45 +27,53 @@
   private final PluginConfigFactory cfgFactory;
   private final String pluginName;
 
-  @Inject
-  HttpModule(PluginConfigFactory cfgFactory,
-      @PluginName String pluginName) {
-    this.cfgFactory = cfgFactory;
-    this.pluginName = pluginName;
-  }
-
```

Comment:
Please revert all formatting changes for the existing code, and do no reformat anything. Pay attention, that the new code is formatted exactly in the same way as the old one.